src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -47,29 +91,45 @@ test_run(void)
 	int			 i;
 	int			 rc = 0;
 
-	if (test_g.t_save_cfg) {
-		rc = crt_group_config_path_set(test_g.t_cfg_path);
-		D_ASSERTF(rc == 0, "crt_group_config_path_set failed %d\n", rc);
-	}
+	if (test_g.t_skip_init) {
+		DBG_PRINT("Skipping init stage.\n");
 
-	tc_cli_start_basic(test_g.t_local_group_name,
-			   test_g.t_remote_group_name,
-			   &grp, &rank_list, &test_g.t_crt_ctx[0],
-			   &test_g.t_tid[0], test_g.t_srv_ctx_num,
-			   test_g.t_use_cfg, NULL);
+	} else if (!test_g.t_skip_init) {
 
-	rc = sem_init(&test_g.t_token_to_proceed, 0, 0);
-	D_ASSERTF(rc == 0, "sem_init() failed.\n");
+		if (test_g.t_save_cfg) {
+			rc = crt_group_config_path_set(test_g.t_cfg_path);
+			D_ASSERTF(rc == 0, "crt_group_config_path_set failed %d\n", rc);
+		}
 
-	/* register RPCs */
-	rc = crt_proto_register(&my_proto_fmt_test_group2);
-	D_ASSERTF(rc == 0, "crt_proto_register() failed. rc: %d\n",
-			rc);
+		tc_cli_start_basic(test_g.t_local_group_name,
+				   test_g.t_remote_group_name,
+					 &grp, &rank_list, &test_g.t_crt_ctx[0],
+					 &test_g.t_tid[0], test_g.t_srv_ctx_num,
+					 test_g.t_use_cfg, NULL);
+
+		rc = sem_init(&test_g.t_token_to_proceed, 0, 0);
+		D_ASSERTF(rc == 0, "sem_init() failed.\n");
+
+		/* register RPCs */
+		rc = crt_proto_register(&my_proto_fmt_test_group2);
+		D_ASSERTF(rc == 0, "crt_proto_register() failed. rc: %d\n",
+			  rc);
+
+		/* Process the --rank option, e.g., --rank 1,2-4 */
+		if (test_g.cg_num_ranks > -1)
+			rank_list = uint32_array_to_rank_list((uint32_t *)test_g.cg_ranks,
+							      test_g.cg_num_ranks);
```

Comment:
(style)  line over 80 characters

Reply:
[Vim `ruler`](http://vimdoc.sourceforge.net/htmldoc/options.html#'ruler') says this line is 72 chars long, and a vim regex doesn't find any code lines (outside of the copyright notice at the top) exceeding 70 chars (`/\v^.{70,}`) in `test_group_np_cli.c`.